lib/ansible/modules/network/aci/aci_aaa_user.py

```diff
@@ -82,6 +82,12 @@
     type: str
     choices: [ absent, present, query ]
     default: present
+  name_alias:
+    version_added: '2.10'
+    description:
+    - nameAlias field to alias the current object.
```

Comment:
The description should be a full sentence. So this probably ought to read:

> The alias for the current object. This relates to the `nameAlias` field in ACI.

Reply:
@nkatarmal-crest